src/exchange/shared-account/add/shared-account-add.controller.js

```diff
@@ -107,7 +107,7 @@ angular.module('Module.exchange.controllers').controller(
         this.$routerParams.organization,
         this.$routerParams.productId,
       ).then((data) => {
-        this.optionsToCreateNewAccounts = data;
+        this.optionsToCreateNewAccounts = _.clone(data);
```

Comment:
```suggestion
        this.optionsToCreateNewAccounts = _.clone(data, true);
```

In case you need deep cloning